test/configuration/configuration.test.ts

```diff
@@ -1,9 +1,11 @@
 import * as assert from 'assert';
 import * as srcConfiguration from '../../src/configuration/configuration';
 import * as testConfiguration from '../testConfiguration';
-import { cleanUpWorkspace, setupWorkspace } from './../testUtils';
+import * as vscode from 'vscode';
+import { cleanUpWorkspace, setupWorkspace, createRandomFile } from './../testUtils';
 import { Mode } from '../../src/mode/mode';
 import { newTest } from '../testSimplifier';
+import { utils } from 'mocha';
```

Comment:
Unnecessary import, I think

Reply:
Good catch, fixed in [1bf1d5c](https://github.com/VSCodeVim/Vim/pull/7747/commits/1bf1d5ce27eaeffbdf7f706f823fc2593506ced4)